makerspace_backend/runserver.py

```diff
@@ -0,0 +1,2 @@
+from makerspace_backend import app
+app.run(debug=True)
```

Comment:
We could make the debug false by default and then set the true as a -d flag in this. We should also wrap this with a 
```python
if __name__ == '__main__':
    ...
```